src/main/kotlin/io/github/anki/anki/controller/ControllerExceptionHandler.kt

```diff
@@ -9,53 +8,54 @@ import org.slf4j.LoggerFactory
 import org.springframework.http.HttpStatus
 import org.springframework.http.ResponseEntity
 import org.springframework.web.HttpRequestMethodNotSupportedException
-import org.springframework.web.bind.MethodArgumentNotValidException
 import org.springframework.web.bind.annotation.ExceptionHandler
 import org.springframework.web.bind.annotation.RestControllerAdvice
+import org.springframework.web.bind.support.WebExchangeBindException
+import reactor.core.publisher.Mono
 
 @RestControllerAdvice
 class ControllerExceptionHandler {
     @ExceptionHandler(HttpRequestMethodNotSupportedException::class)
-    fun methodNotSupportedHandler(): ResponseEntity<String> =
-        ResponseEntity.status(HttpStatus.NOT_FOUND).body("Not Found")
+    fun methodNotSupportedHandler(): Mono<ResponseEntity<String>> =
```

Comment:
is it really required to wrap the response entity into `mono`?

Not sure, but I guess it would work without the wrapping